ivy/stateful/module.py

```diff
@@ -1210,7 +1210,10 @@ def _replace_update_v(self, new_v, native=None):
                 native.__setattr__(k, v)
             elif isinstance(v, torch.Tensor):
                 # noinspection PyProtectedMember
-                native.__setattr__(k, torch.nn.Parameter(v))
```

Comment:
I think we could just replace [this](https://github.com/unifyai/ivy/pull/25713/files#diff-9d3cd8360ac6368fb7cfebdd75ef04230b2963749af122e8515e44ea6ddf3971L1213) line with `native.__setattr__(k, torch.nn.Parameter(v, requires_grad=v.requires_grad))` instead and remove the `if-else` added.
@hmahmood24 and @mattbarrett98 could you please confirm if this change to the converters causes any issues in the transpiler? Thanks!

Reply:
changed it. That also works!